apps/backoffice-v2/src/pages/Home/components/RiskAndAlertsOverviewSection/RiskAndAlertsOverviewSection.tsx

```
@@ -0,0 +1,113 @@
+          }}
+          nameKey="riskLevel"
+          config={activeCasesByRiskLevelData.reduce((acc, curr) => {
+            const riskLevelMap = {
+              Low: { color: '#4CAF50' },
+              Medium: { color: '#FFB74D' },
+              High: { color: '#FF9800' },
+              Critical: { color: '#F44336' },
+            };
+
+            return {
+              ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)